js/okcoin.js

```diff
@@ -2155,8 +2154,7 @@ module.exports = class okcoin extends Exchange {
                     } else {
                         notional = (notional === undefined) ? amount : notional;
                     }
-                    const precision = market['precision']['price'];
-                    request['notional'] = this.decimalToPrecision (notional, TRUNCATE, precision, this.precisionMode);
+                    request['notional'] = this.decimalToPrecision (notional, TRUNCATE, market['precision']['price'], this.precisionMode);
```

Comment:
Let's use `this.costToPrecision` in cases like this:

```suggestion
                    request['notional'] = this.costToPrecision (symbol, notional);
```